mayland/inc/wpcom-colors.php

```diff
@@ -145,7 +205,7 @@
 			.wp-block-newspack-blocks-homepage-articles article .entry-title a:active,
 			.wp-block-newspack-blocks-homepage-articles article .entry-title a:focus,
 			.wp-block-newspack-blocks-homepage-articles article .entry-title a:hover,
-			a:not(.has-text-color),
+			a,
```

Comment:
This is the only line that changed, the rest of the changes are linter fixes.

Reply:
Do the linting fixes have to be in there?
With such a small change it's a shame not to play golf with the DIFF to make the history clear.

Reply:
They don't! I do like golf.